code-challenges.test.js

```diff
@@ -15,35 +15,98 @@
-
+describe("shuffledArray", () => {
+ it("takes in an array, removes the first item from the array and shuffles the remaining content", () => {
+   const colors1 = ["purple", "blue", "green", "yellow", "pink"]
+   // Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
+   const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
+   // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+   expect(shuffledArray(colors1)).toEqual(expect.arrayContaining(["yellow", "blue", "pink", "green"]))
+   expect(shuffledArray(colors2)).toEqual(expect.arrayContaining(["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]))
+ })
+})
+
+// ● shuffledArray › takes in an array, removes the first item from the array and shuffles the remaining content
+//
+//   ReferenceError: shuffledArray is not defined
 
 // b) Create the function that makes the test pass.
 
+//Create a function - shuffledArray
+//Parameter - array
+//Use the .slice() method to take remove the first item from the array
+//Use the .sort() method followed by Math.random() to randomize the strings in the sliced array.
+
+const shuffledArray = (array) => {
+  let sliced = array.slice(1)
+  return sliced.sort(() => .5 - Math.random());
+}
 
 // --------------------2) Create a function that takes an array of numbers and returns an array of the minimum and maximum numbers in that order.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const nums1 = [3, 56, 90, -8, 0, 23, 6]
-// Expected output: [-8, 90]
-const nums2 = [109, 5, 9, 67, 8, 24]
-// Expected output: [5, 109]
-
+describe("minAndMax", () => {
+  it("takes an array of numbers and returns an array of the minimum and maximum numbers in that order", () => {
+    const nums1 = [3, 56, 90, -8, 0, 23, 6]
+    // Expected output: [-8, 90]
+    const nums2 = [109, 5, 9, 67, 8, 24]
+    // Expected output: [5, 109]
+    expect(minAndMax(nums1)).toEqual([-8, 90])
+    expect(minAndMax(nums2)).toEqual([5, 109])
+  })
+})
+
+// ● minAndMax › takes an array of numbers and returns an array of the minimum and maximum numbers in that order
+//
+//    ReferenceError: minAndMax is not defined
 
 // b) Create the function that makes the test pass.
 
+//Create a function - minAndMax
+//Parameter - array
+//Create an empty array to hold min and max
+//Create min and max variable, use .push() method to have the output into new empty array
+//Use Math.min.apply and Math.max.apply to have it take in array elements
+//Sort the min and max numbers
+//Return the new array with just the two filtered numbers
+
+const minAndMax = (array) => {
+  let total = []
+  let min = total.push(Math.min.apply(Math, array))
+  let max = total.push(Math.max.apply(Math, array))
+  return total.sort(function (a,b){return a-b})
+}
+
```

Comment:
Just for future notes, I recommend you including an explanation of why you had to also use .apply with Math.min and Math.max